Client/Frontend/Widgets/AutocompleteTextField.swift

```diff
@@ -122,13 +129,21 @@ class AutocompleteTextField: UITextField, UITextFieldDelegate {
         super.setMarkedText(markedText, selectedRange: selectedRange)
     }
 
-    override func insertText(text: String) {
-        removeCompletion()
-        super.insertText(text)
+    func SELtextDidChange(textField: UITextField) {
+        //If character is deleted than we want to return do not want autocompletion.
+        if characterDeleted {
+            characterDeleted = false
+            
+            return
```

Comment:
If any character is deleted than we simply returns. We do not want autocompletion.